### PR TITLE
Updates installer from review and bug fix

### PR DESCRIPTION
**Pull request by jmckulk**
This commit adds documentation on how to update the installer service account to
user image pull secrets. It also adds new inventory parameter to defaults in the
`pgo-install.sh` script. This new value was added to the inventory file and
ansible defaults but not here. This change syncs the two defaults. The parameter
lists are reordered to be alphabetical.

Condenses the install, update, and uninstall jobs to a `deploy.yml` file. This file can be updated to run each of these jobs. The docs are updated to specify that a cleanup is needed after each job is run. 

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable? I was able to create an image pull secret, link it to the service account, and deploy the installer. Also tested by following the docs to pull an image from the crunchy private registry

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**

**What is the new behavior (if this is a feature change)?**

**Other information**:
[ch7740]
[ch7759]

### CODE REVIEW

File: installers/method/kubectl/uninstall.yml [hunk 1]
```diff
@@ -107,15 +111,9 @@ spec:
                     value: "gce"
                   - name: REPLICA_STORAGE
                     value: "gce"
-                  - name: PGBADGERPORT
```
Comment (jmckulk) [inline installers/method/kubectl/uninstall.yml:110]: was removed and needs to be added back in each file

File: docs/content/Installation/postgres-operator-installer/kubectl-install.md [hunk 2]
```diff
@@ -37,6 +37,42 @@ cluster-admin role. This is required for the installer to run correctly. If you
 have a preconfigured service account with the cluster-admin role, you can remove
 this section of the yaml and update the service account name in the job spec.
 
+##### Image Pull Secrets
+
+If you are pulling the Postgres-Operator images from a private registry you will
```
Comment (jkatz) [inline docs/content/Installation/postgres-operator-installer/kubectl-install.md:42]: PostgreSQL Operator

File: docs/content/Installation/postgres-operator-installer/kubectl-install.md [hunk 3]
```diff
@@ -37,6 +37,42 @@ cluster-admin role. This is required for the installer to run correctly. If you
...
+need to setup an `imagePullSecret` with access to the registry. The image pull
```
Comment (jkatz) [inline docs/content/Installation/postgres-operator-installer/kubectl-install.md:43]: [`imagePullSecret`](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)

File: docs/content/Installation/postgres-operator-installer/kubectl-install.md [hunk 4]
```diff
@@ -37,6 +37,42 @@ cluster-admin role. This is required for the installer to run correctly. If you
...
+secret will need to be added to the installer service account to have access.
+The secret will need to be created in each namespace that the Postgres-Operator
```
Comment (jkatz) [inline docs/content/Installation/postgres-operator-installer/kubectl-install.md:45]: PostgreSQL Operator

**Pull request merged**